src/Microsoft.Azure.SignalR.Common/ServiceConnections/WeakServiceConnectionContainer.cs

```diff
@@ -47,6 +47,8 @@ public override Task HandlePingAsync(PingMessage pingMessage)
             return Task.CompletedTask;
         }
 
+        public override Task ShutdownAsync(TimeSpan timeout) => StopAsync();
```

Comment:
>StopAsync [](start = 64, length = 9)

I am a little confused about the naming now....

Reply:
There is nothing to do with the `WeekConnection` while shutting down it (no active client connection attached to it), so we simply reuse the `StopAsync`.